modules/arcgis/hooks/useArcgisData.ts

```diff
@@ -49,16 +50,13 @@ export const useArcgisData = () => {
   })
 
   return {
-    rawAssistantsData: rawAssistantsData as FeatureCollection<
-      Point,
-      MapInterestPoint & { web: string }
-    >,
-    rawParkomatsData: rawParkomatsData as FeatureCollection<Point, ParkomatPoint>,
-    rawPartnersData: rawPartnersData as FeatureCollection<Point, PartnerPoint>,
-    rawParkingLotsData: rawParkingLotsData as FeatureCollection<Point, ParkingLotPoint>,
-    rawBranchesData: rawBranchesData as FeatureCollection<Point, BranchPoint>,
-    rawUdrData: rawUdrData as FeatureCollection<Polygon, MapUdrZone>,
-    rawOdpData: rawOdpData as FeatureCollection<Polygon, GeoJsonProperties>,
-    rawZonesData: rawZonesData as FeatureCollection<Polygon, GeoJsonProperties>,
+    rawAssistantsData: rawAssistantsData as FeatureCollection<Point, AssistantPoint> | undefined,
+    rawParkomatsData: rawParkomatsData as FeatureCollection<Point, ParkomatPoint> | undefined,
+    rawPartnersData: rawPartnersData as FeatureCollection<Point, PartnerPoint> | undefined,
+    rawParkingLotsData: rawParkingLotsData as FeatureCollection<Point, ParkingLotPoint> | undefined,
+    rawBranchesData: rawBranchesData as FeatureCollection<Point, BranchPoint> | undefined,
+    rawUdrData: rawUdrData as FeatureCollection<Polygon, MapUdrZone> | undefined,
+    rawOdpData: rawOdpData as FeatureCollection<Polygon, GeoJsonProperties> | undefined,
+    rawZonesData: rawZonesData as FeatureCollection<Polygon, GeoJsonProperties> | undefined,
```

Comment:
I'm nervous about this whole custom typing :D

Reply:
We will not be using that hook anymore anyway